stable/prometheus/templates/server-service.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       port: {{ .Values.server.service.servicePort }}
       protocol: TCP
       targetPort: 9090
+{{- if .Values.server.service.nodePort }}
+      nodePort: {{ .Values.server.service.nodePort }}
+{{- end }}
```

Comment:
Nit: indent the if...end statements 4 spaces so the dashes align with the 'n'  in `nodePort`